sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/LiteralsShouldNotBePassedAsLocalizedParameters.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.ComponentModel;
+
+namespace Tests.Diagnostics
+{
+    class Program

```

Comment:
Could you test with resx usage (one with classic way `MyResx.MyKey` and the other with the `ResourceManager`)? Also could you make some tests with the usage of a constant field and a constant local variable.